src/components/attendee/InputSelect/InputSelect.module.scss

```diff
@@ -28,5 +32,29 @@
   box-sizing: border-box;
   width: 20%;
   border: none;
-  border-right: 5px solid transparent;
+  border-right: border.$radius--xs solid transparent;
+}
+
+.errorIcon {
+  background-image: url(/assets/icons/input-error-icon.svg);
+  height: $-error-icon-size;
+  width: $-error-icon-size;
```

Comment:
[NOTE] We might want to re-introduce `square-size` function some day if these keep popping up.

Reply:
yep, we might need it